relay-server/src/metrics_extraction/transactions/mod.rs

```diff
@@ -725,6 +735,7 @@ mod tests {
                     "environment": "fake_environment",
                     "geo.country_code": "US",
                     "release": "1.2.3",
+                    "sentry.score_profile_version": "alpha",
```

Comment:
Please make sure to register the new tag name [here](https://github.com/getsentry/sentry/blob/master/src/sentry/sentry_metrics/indexer/strings.py) before merging this PR.

Reply:
Added it here: https://github.com/getsentry/sentry/pull/66795

Reply:
Could you link that PR from the description? Thanks beforehand!